File: kata-solutions/video-store-kata/video-store-java/src/main/java/ChildrensMovie.java
package com.cleancode.martinfowler.videostore;

public class ChildrensMovie extends Movie {
  public ChildrensMovie(String title) {
    super(title);
  }

  @Override
  public double determineAmount(int daysRented) {
    double thisAmount = 1.5;
    if (daysRented > 3)
      thisAmount += (daysRented - 3) * 1.5;

    return thisAmount;
  }

  @Override
  public int determineFrequentRenterPoints(int daysRented) {
    return 1;
  }
}

File: kata-solutions/video-store-kata/video-store-python/src/childrens_movie.py
from .movie import Movie


class ChildrensMovie(Movie):
    def __init__(self, title):
        super().__init__(title)

    def calculate_amount(self, days_rented):
        amount = 1.5
        if days_rented > 3:
            amount += (days_rented - 3) * 1.5
        return amount

    def calculate_frequent_renter_points(self, days_rented):
        return 1
